roles/provision-mdc-server-apb/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 # MCS Values
 mobile_developer_console_image: "docker.io/aerogear/mobile-developer-console"
-mobile_developer_console_image_tag: "latest"
+mobile_developer_console_image_tag: "master"
```

Comment:
found out that `latest` tag [is not the one that's up-to-date](https://hub.docker.com/r/aerogear/mobile-developer-console/tags/)